src/components/DatePicker/index.js

```diff
@@ -93,14 +91,12 @@ DatePicker.propTypes = {
   showTimeSelect: PropTypes.bool,
   label: PropTypes.string,
   disabled: PropTypes.bool,
-  value: PropTypes.object,
-  placeholder: PropTypes.string
+  value: PropTypes.object
 };
 
 DatePicker.defaultProps = {
   onChange: () => {},
-  minDate: tomorrow,
-  placeholder: 'enter date...'
```

Comment:
Why remove placeholder? I imagine this doesn't even function? :)

Reply:
The DatePicker always has a default value, and the user can only go forward or backward, both on mobile and PC. Since placeholder is only shown when the input is empty, it would never show!

Reply:
nice